downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -44,6 +46,52 @@ enum class VcsHost(
      */
     vararg supportedTypes: VcsType
 ) {
+    AZURE_DEVOPS("dev.azure.com", VcsType.GIT) {
+        private val commitPrefix = "GC"
```

Comment:
Maybe name this `gitCommitPrefix`, as I guess that's what "GC" (= "Git Commit") stands for?